ethers-core/src/types/trace/filter.rs

```diff
@@ -266,6 +266,32 @@ pub enum RewardType {
     External,
 }
 
+/// Basic trace type for the Geth client.
+#[derive(Debug, Clone, PartialEq, Deserialize, Serialize)]
+pub struct GethTrace {
+    failed: bool,
+    gas: i64,
+    #[serde(rename = "returnValue")]
+    return_value: Option<Bytes>,
+    #[serde(rename = "structLogs")]
+    struct_logs: Vec<OpCodeLog>,
+}
+
+/// OpCodeLog type for the Geth trace.
+#[derive(Debug, Clone, PartialEq, Deserialize, Serialize)]
+pub struct OpCodeLog {
+    depth: u64,
+    error: Option<String>,
+    gas: u64,
+    #[serde(rename = "gasCost")]
+    gas_cost: u64,
+    memory: Option<Bytes>,
+    op: String,
+    pc: usize,
+    stack: Option<Vec<U256>>,
+    storage: Option<BTreeMap<H256, H256>>,
```

Comment:
I'd prefer to use default her for both stack and storage, getting rid of the option